pkg/types/azure/machinepool.go

```diff
@@ -72,11 +72,8 @@ type MachinePool struct {
 	Settings *SecuritySettings `json:"settings,omitempty"`
 
 	// Identity is the type of identity used for the virtual machine.
-	// The type 'SystemAssigned' is an implicitly created identity.
-	// The generated identity will be assigned a Subscription contributor role.
 	// The type 'UserAssigned' is a standalone Azure resource provided by the user
 	// and assigned to the VM
-	// Identity can only be set for control-plane nodes.
```

Comment:
force pushed to update (remove) this inaccurate line.

Reply:
Need another rebase I think :sweat_smile: